package.json

```diff
@@ -64,7 +64,6 @@
     "@babel/core": "^7.20.12",
     "@babel/eslint-parser": "^7.19.1",
     "@babel/preset-react": "7.18.6",
-    "@strapi/admin-test-utils": "workspace:*",
```

Comment:
how come you've edited this?

Reply:
Docker stuff to test it in kubernetes... I will put it back in the next commit but this is draft yet (I forgot to set it, my fault)

Reply:
awesome, thank you!